### ISSUE TITLE
Sphenoid wing dysplasia

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#10696]
**Preferred term label:**
Sphenoid wing dysplasia

**Synonyms**

**Definition (free text, please give PubMed ID)**
Hypoplasia or aplasia of the greater or lesser wing of the sphenoid bone, typically resulting in widening of the superior orbital fissure, elevation of the [lesser sphenoid wing, and ipsilateral orbital enlargement.

Comment: All reported cases of congenital sphenoid wing dysplasia have been attributed to neurofibromatosis type 1 (NF1). Sphenoid bone dysplasia occurs in 3–11 percent of the subjects with neurofibromatosis type 1 NF1 patients. This malformation, a distinctive diagnostic feature for NF1, is characterized by progressive proptosis and facial disfiguration secondary to herniation of meningeal and cerebral structures. Unilateral in most of the cases, sphenoid bone dysplasia associated with plexiform neurofibroma in the territory of the trigeminal nerve in several cases and pooling of cerebrospinal fluid (CSF) in all cases often described in the past as an arachnoid cyst.
PMID:28409229
PMID:33758729
PMID:4980937
**Parent term (use hpo.jax.org/app)**
Abnormal facial skeleton morphology HP:0011821

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
NF1

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
Adding new term: 
- ID: HP:6001124
- label: Sphenoid wing dysplasia
- parents: [Abnormal facial skeleton morphology (HP:0011821)](https://hpo.jax.org/app/browse/term/HP:0011821)
- pmid: PMID:28409229|PMID:33758729|PMID:4980937
- def: Hypoplasia or aplasia of the greater or lesser wing of the sphenoid bone, typically resulting in widening of the superior orbital fissure, elevation of the [lesser sphenoid wing, and ipsilateral orbital enlargement.
- comment: All reported cases of congenital sphenoid wing dysplasia have been attributed to neurofibromatosis type 1 (NF1). Sphenoid bone dysplasia occurs in 3-11 percent of the subjects with neurofibromatosis type 1 NF1 patients. This malformation, a distinctive diagnostic feature for NF1, is characterized by progressive proptosis and facial disfiguration secondary to herniation of meningeal and cerebral structures. Unilateral in most of the cases, sphenoid bone dysplasia associated with plexiform neurofibroma in the territory of the trigeminal nerve in several cases and pooling of cerebrospinal fluid (CSF) in all cases often described in the past as an arachnoid cyst.
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/10696
- synonyms:
